BASIQ-65: Remove Redux and use Context

albatahmet opened the pull request:
First context created and transaction invocation moved to context side
work will continue...

albatahmet committed:
BASIQ-65: Remove Redux and use Context

First context created and transaction invocation moved to context side

albatahmet committed:
Basiq-65: remove redux and use context

The transactionAction page has been removed, the code has been improved and the queries to be used have been removed

k-cem-fd commented on .env.sample:
```diff
@@ -1 +0,0 @@
-BASIQ_API_KEY=
```
This file is a sample to create new env file and put api key into it. Please re-add @albatahmet

albatahmet replied:
I re-added this file

k-cem-fd commented on components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js:
```diff
@@ -1,31 +1,28 @@
-  const { isCompleted, transactionsError } = useSelector(state => state.userTransactions);
+  const transactionContext = useTransactionsDataContext();
+  const isCompleted = transactionContext.state.isCompleted;
+  const transactionsError = transactionContext.state.transactionsError;
```
Can you update as below or similar ? 
_const { isCompleted, transactionsError } = transactionContext.state;_

albatahmet replied:
updated to a similar structure

k-cem-fd commented on store/context/logger/index.js:
```diff
@@ -0,0 +1,29 @@
+
+	return [state, dispatchWithLogger]
+}
+
```
Can you tidy this code using linter ?

albatahmet replied:
I edited it the way you said, is it correct?

k-cem-fd replied:
Yeah, perfect 👍

k-cem-fd commented on store/context/transactionContext.js:
```diff
@@ -0,0 +1,93 @@
+export const TransactionsDataContext = createContext();
+export const useTransactionsDataContext = () => useContext(TransactionsDataContext);
+
+///gerekli ilk stateler buraya yazılacak
```
Remove these comments or write in English if they are necessary

albatahmet replied:
I removed all comment lines

albatahmet committed:
BASIQ-65: Remove Redux and use Context

Correction of incorrect fields in Merge request.
-- fix the definition structure
-- comment lines removed
--.env.sample file added again
-- edited with logger linter
